README.md

```diff
@@ -4,17 +4,6 @@ QC metrics for BAM files
 
 ## Overview
 
```

Comment:
Overview section is now empty -- can remove the header if nothing is there

Reply:
done

Reply:
I added back the overview contents.